packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -724,36 +734,63 @@ export class BackgroundScript {
     }
 
     const receiver = stream.getPaymentPointer()
-    const { assetCode, assetScale, exchangeRate } = stream.getAssetDetails()
-    const amount = Math.floor(1e9 * exchangeRate).toString() // 1 USD, assetScale = 9
 
     try {
       this.log(`sendTip: sending tip to ${receiver}`)
-      const result = await this.client.query({
+      const result = await this.client.query<SendTipResult>({
         query: `
-          mutation sendTip($receiver: String!) {
-            sendTip(receiver: $receiver) {
+          mutation sendTip($receiver: String!, $amount: Int, $webMonetizationId: String) {
+            sendTip(receiver: $receiver, amount: $amount, webMonetizationId: $webMonetizationId) {
               success
+              amount
+              currency
+              scale
+              receipts
+              destinationAmount {
+                amount
+                assetCode
+                assetScale
+              }
             }
           }
         `,
         token,
         variables: {
-          receiver
+          receiver,
+          amount: undefined, // default
+          webMonetizationId: streamId
```

Comment:
Would `streamId` be the same `requestId` used for web monetization's `Web-Monetization-Id`?

Reply:
Yep, streamId is what will be used for WMID and the requestId in the events